plugins/inspect/config_files_inspector.rb

```diff
@@ -47,9 +47,9 @@ def packages_with_config_files(system)
   end
 
   # returns a hash with entries for changed config files
-  def config_file_changes(system, pkg)
+  def config_file_changes(pkg)
```

Comment:
Assignment Branch Condition size for config_file_changes is too high. [15.78/15]